### PR TITLE
Deployment script for DEX related contracts

### PR DESCRIPTION
**Pull request by HyunggyuJang**
Since there are some occasions that we need to redeploy contracts to reflect any bug fixes or due to runtime issue, it would be better to provide a way to automate deployment process. 

This PR handles it. Note that the tests don't work currently; I'll fix those (migrate to weight V2, compatible with swanky node from https://github.com/AstarNetwork/swanky-node/commit/2334dadf4420d237bf7c2f0d19648122721d09a9) in an upcoming, separate PR.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+PRIVATE_KEY=
```
Comment (PierreOssun) [inline .env.example:1]: Can you add a dummy private key ? to know the format

File: patches/@supercolony+typechain-polkadot+0.5.0.patch [hunk 2]
```diff
@@ -0,0 +1,211 @@
+diff --git a/node_modules/@supercolony/typechain-polkadot/.DS_Store b/node_modules/@supercolony/typechain-polkadot/.DS_Store
```
Comment (PierreOssun) [inline patches/@supercolony+typechain-polkadot+0.5.0.patch:1]: What is this new patch solving ?

**Review by PierreOssun — APPROVED** [review]

File: patches/@supercolony+typechain-polkadot+0.5.0.patch (hunk 2 above)
Comment (HyunggyuJang) [reply patches/@supercolony+typechain-polkadot+0.5.0.patch:1]: This part has nothing to do with my intention; just slipped into it. Will fix it. For the other patches (following diffs) are to handle weight v2.

File: .env.example (hunk 1 above)
Comment (HyunggyuJang) [reply .env.example]: Added: https://github.com/AstarNetwork/wasm-showcase-dapps/pull/40/commits/dd49bb6d6e7622df130e1f1681e7887cf4848832

**Pull request merged**